starlark-lsp/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I will update the READ.me to be more professional in a separate PR; will mention about it being a fork from starlark-lsp and remove the references from it as well. 

I will change the folder name and everything to `klsp` -- unless you different suggestion?

Reply:
There's another alternative where I can have `lsp` in a different repo and use it as package in `cli/cli/lsp.go`; If you think that's better way to organize the code - i can do that too.

Reply:
What's the cost of having it in a separate repo? (this includes future costs as well, in case there are some). I'd personally bias toward extracting it in its own repo if we can. It's quite far from the core business of what the CLI is currently doing, and I'm usually in favour of splitting responsibility clearly when it does not incur additional complexities.
(you can track moving it to another repo in a FLUP, if merging it here gives you velocity, then I'm good with temporarily having it here)

Reply:
if `starlark-lsp` in another repo is that we will be using circleci with `tilt-releaser` in two repos ( and will executed twice) so there maybe additional cost. However, I am not able to determine the cost delta yet, should be slight increase though. Other than that, i do not think there is another major issue.